xarray/tests/test_groupby.py

```diff
@@ -808,6 +808,29 @@ def test_groupby_math_more() -> None:
         ds + ds.groupby("time.month")
 
 
+def test_groupby_math_bitshift() -> None:
+    ds = Dataset(
+        {
+            "x": ("level", np.ones(4, dtype=int)),
+            "y": ("level", np.ones(4, dtype=int) * -1),
+            "level": [0, 1, 2, 3],
```

Comment:
```suggestion
            "level": [0, 1, 2, 2],
```

So that we actually test the groupby  bit.

Reply:
ah yeah, good point.  Had to shuffle it a little bit more but think that covers the actual groupby operation better now.